lib/house.rb

```diff
@@ -0,0 +1,55 @@
+class House
+  attr_reader :address, :rooms
+
+  def initialize(price, address)
+    @price = price
+    @address = address
+    @rooms = []
+  end
+
+  def price
+    @price[1..-1].to_i
+  end
+
+  def add_room(room)
+    @rooms << room
+  end
+
+  def above_market_average?
+    if price >= 500000
```

Comment:
We can simplify this to `price > 50000`. No need for the ifs. Also, pay attention to detail. We want a `>` not a `>=`.